src/components/CompletingParty/package.json

```diff
@@ -0,0 +1,11 @@
+{
+  "name": "@bcrs-shared-components/completingParty",
+  "version": "0.0.0",
+  "publishConfig": {
+    "access": "public"
+  },
+  "dependencies": {
+    "@bcrs-shared-components/interfaces": "1.0.33",
+    "vue-property-decorator": "^8.4.0"
+  }
```

Comment:
Is lodash a dependency?

Reply:
Also vuelidate.

Reply:
The root dependencies are for this project as a whole, ie, Storybook.

The local dependencies call out what this particular component needs, which is probably already being provided by the parent project but I think it's still nice to call it out to make this component self-contained / independent of external hidden dependencies.